tools/run_tests.sh

```diff
@@ -39,7 +39,7 @@ set_sanitizer_options() {
         echo " [i] Leaksanitizer disabled"
         ASAN_OPTIONS=detect_leaks=0
     fi
-    ASAN_OPTIONS=$ASAN_OPTIONS:detect_stack_use_after_return=1:detect_stack_use_after_scope=1:check_initialization_order=true:strict_init_order=true:new_delete_type_mismatch=0:suppressions=$BASE_DIR/sanitizer_blacklist/asan_runtime.txt
+    ASAN_OPTIONS=$ASAN_OPTIONS:detect_stack_use_after_return=1:detect_stack_use_after_scope=1:check_initialization_order=true:strict_init_order=true:new_delete_type_mismatch=0:suppressions=$BASE_DIR/sanitizer_blacklist/asan_runtime.txt:intercept_tls_get_addr=0
```

Comment:
Before I can approve this could you please:
1. provide me with a link which describes what the option `intercept_tls_get_addr` does 
2. explain why we require it?